### PR TITLE
Ft refactor

### PR DESCRIPTION
**Pull request by AnthoniaOkafor**
## Description
In this task, I renamed the project called goal3 to project in order to indicate it is the project folder. I also moved blog app into projectapps folder which is the folder created for apps

All pull requests should be related to open issues. Indicate the issue(s) bellow and remove this line:
Fixes #(issue number 1) #(issue number 2 if applicable)

## How Has This Been Tested?
Please describe in detail how you tested your changes.
Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code etc.

## Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added necessary inline code documentation
- [ ] I have added tests that prove my fix is effective and that this feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by happyjosh-tech** [comment]
Please can you add screenshot of the new interface, incident, responders, etc

**Comment by happyjosh-tech** [comment]
you made your commit into master branch instead of develop branch

File: src/Zoit/Zoitapp/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class ZoitappConfig(AppConfig):
+    name = 'Zoitapp'
```
Comment (patrickRobotics) [inline src/Zoit/Zoitapp/apps.py:5]: use this name as the name of the installed app in settings.py

File: src/Zoit/zoit/settings.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+INSTALLED_APPS = [
+    'Zoitapp.apps.ZoitappConfig',
```
Comment (patrickRobotics) [inline src/Zoit/zoit/settings.py:36]: use the name used in the class, i.e. name='Zoitapp', and not the classname for clarity
so here it should be:
INSTALLED_APPS = [
 'Zoitapp',
...,
]
Preferably add it at the end of the dictionary.

File: src/project/settings.py [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'rest_framework',
+    'bootstrap4',
+    'bootstrap_datepicker_plus',
+    'tz_detect',
+
+    #apps should be defined here
+    'projectapps.usersapp',
```
Comment (patrickRobotics) [inline src/project/settings.py:54]: if all went well, there is no need of prefixing with `projectapps` as django will locate all apps in the specific app's `apps.py` file

**Review by patrickRobotics — COMMENTED** [review]
Left some comments

**Pull request closed without merging**